apis/ec2/manualv1alpha1/referencers.go

```diff
@@ -45,3 +46,52 @@ func (mg *VPCCIDRBlock) ResolveReferences(ctx context.Context, c client.Reader)
 	mg.Spec.ForProvider.VPCIDRef = rsp.ResolvedReference
 	return nil
 }
+
+// ResolveReferences of this VPCCIDRBlock
+func (mg *Instance) ResolveReferences(ctx context.Context, c client.Reader) error {
+	r := reference.NewAPIResolver(c, mg)
+
+	// Resolve spec.forProvider.securityGroupIds
+	rgids, err := r.ResolveMultiple(ctx, reference.MultiResolutionRequest{
+		CurrentValues: mg.Spec.ForProvider.SecurityGroupIDs,
+		References:    mg.Spec.ForProvider.SecurityGroupIDRefs,
+		Selector:      mg.Spec.ForProvider.SecurityGroupIDSelector,
+		To:            reference.To{Managed: &v1beta1.SecurityGroup{}, List: &v1beta1.SecurityGroupList{}},
+		Extract:       reference.ExternalName(),
+	})
+	if err != nil {
+		return errors.Wrap(err, "spec.forProvider.securityGroupIds")
+	}
+	mg.Spec.ForProvider.SecurityGroupIDs = rgids.ResolvedValues
+	mg.Spec.ForProvider.SecurityGroupIDRefs = rgids.ResolvedReferences
```

Comment:
Same question as above. Confirming we want to allow Refs/Selectors for both "SecurityGroups" and "SecurityGroupIDs"

Reply:
For context:

Current AWS users are probably using either the "SecurityGroup" or "SecurityGroupId" field, so it makes sense to support both. But for Refs and Selectors within the provider, I believe the current SecurityGroup Resource already supplies the ID as an external-name. So we would only need one Ref/Selector to find it.